### PR TITLE
Fix CIs

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ dependencies:
 - mkl
 - numpy
 - scipy
-- matplotlib
+- matplotlib<=3.0
```
Comment (larsoner) [inline environment.yml]: IIUC 3.0.0 is the problem

File: environment.yml [hunk 2]
```diff
@@ -7,10 +7,10 @@ dependencies:
...
 - cython
 - pyqt>=5.9
-- vtk>=8
```
Comment (larsoner) [inline environment.yml]: And what does this help?

Comment (massich) [reply environment.yml]: I was trying to find if it was the only one having issues with python 2.7 for in appveyor. 'Cos I can build the conda enviroment in my machine. So it was just some messing arround.

Comment (larsoner) [reply environment.yml]: Ahh I see. I think you might be right, so I got rid of the `matplotlib` one and we can see if AppVeyor is happy

**Comment by larsoner** [comment]
Argh no still broken
```
UnsatisfiableError: The following specifications were found to be in conflict:
  - pyqt[version='>=5.9'] -> python[version='>=3.5,<3.6.0a0']
  - python=2.7
Use "conda info <package>" to see the dependencies for each package.
```

**Pull request closed without merging**